README.md

```diff
@@ -294,7 +306,19 @@ const fileBox = FileBox.fromRemote(
 console.log(fileBox.name) // Output: file-box-logo.jpg
 ```
 
-#### 3.2 `metadata: Metadata { [key: string]: any }`
+#### 3.2 `md5`
+
+File md5 of the file in the box. The value is set by user, not computed from file.
+
+```ts
+const fileBox = FileBox.fromUrl(
+  'https://huan.github.io/file-box/images/file-box-logo.jpg',
+  {md5: 'some-random-md5'}
```

Comment:
The name here is confusing: `some-random-md5`.

It is not random, it's calculated by the MD5 alghorithem.

Please rename it to a better one.